Src/CSharpier/SyntaxPrinter/SyntaxNodePrinters/InitializerExpression.cs

```diff
@@ -37,8 +37,7 @@ private static Doc Print(InitializerExpressionSyntax node, string? groupId)
                 is not (ObjectCreationExpressionSyntax
                     or ArrayCreationExpressionSyntax
                     or ImplicitArrayCreationExpressionSyntax)
-            && node.Kind()
-                is not SyntaxKind.WithInitializerExpression ? Doc.Group(result) : result;
+            && node.Kind() is not SyntaxKind.WithInitializerExpression ? Doc.Group(result) : result;
```

Comment:
What's going on here? Is this below the line length or do we have a bug?

Also, I think we should definitely break/indent the ternary operator if the condition needed a break.